### PR TITLE
[TEVA-2642] fix layout of dashboard header

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2642

### CODE REVIEW

File: app/components/shared/dashboard_component/dashboard_component.scss [hunk 1]
```diff
@@ -9,25 +9,19 @@
...
+      display: flex;
     }
 
     &__button {
-      bottom: govuk-spacing(3);
-      margin-top: 0;
-      position: absolute;
-      right: 0;
+      flex-grow: 1;
+      text-align: right;
```
Comment (cpjmcquillan) [inline app/components/shared/dashboard_component/dashboard_component.scss:17]: can you add a `margin-bottom` to the button?

Comment (alexbowen) [reply app/components/shared/dashboard_component/dashboard_component.scss:17]: i just pushed a different chnage for this

Comment (alexbowen) [reply app/components/shared/dashboard_component/dashboard_component.scss:17]: removing the top margin below brings this into alignment

Comment (cpjmcquillan) [reply app/components/shared/dashboard_component/dashboard_component.scss:17]: I think on the template there are margin top/bottom 0 classes, so might need to change those

Comment (alexbowen) [reply app/components/shared/dashboard_component/dashboard_component.scss:17]: they are needed

Comment (alexbowen) [reply app/components/shared/dashboard_component/dashboard_component.scss:17]: the margin top one can be removed and i removed it

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**